### DISCUSSION TITLE
How to add new countries?

### DISCUSSION THREAD

**Discussion by afe038** [pimcore/pimcore#9914] [General]
We noticed that Kosovo is not present on the countries list. How can we add new countries so that they can be present on the country/multi-country selection?

**Comment by NiklasBr** [comment]
Pimcore is using [umpirsky/country-list](https://github.com/pimcore/pimcore/blob/10.x/composer.json#L146) and it looks like Kosovo is missing in some of those files, for example: https://github.com/umpirsky/country-list/blob/master/data/en/country.php

You could ask them to add, or create a PR there perhaps.
